hatchet_sdk/worker/runner/run_loop_manager.py

```diff
@@ -39,7 +39,7 @@ def __post_init__(self):
         self.start()
 
     def start(self, retry_count=1):
-        k = self.loop.create_task(self.async_start(retry_count))
+        self.loop.create_task(self.async_start(retry_count))
```

Comment:
i think we had some issues with "no ref to future" without the `k` and this at least silenced it... not sure what the right way of doing this is/if its not an issue